### PR TITLE
new package: pocketbase

### PR DESCRIPTION
**Pull request by SunPodder**

### CODE REVIEW

File: packages/pocketbase/build.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	export GOBUILD=CGO_ENABLED=0
+	
+	go build \
+		-trimpath \
+		-o "pocketbase.bin" \
+		main.go
+}
+
+termux_step_make_install() {
+    install -m700 examples/base/pocketbase.bin "${TERMUX_PREFIX}"/bin/pocketbase
```
Comment (Grimler91) [inline packages/pocketbase/build.sh:28]: Please use tabs for indentation, and add a newline at the the end

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/pocketbase/build.sh (hunk 1 above)
Comment (SunPodder) [reply packages/pocketbase/build.sh:28]: Is that strictly required? I don't usually use vscode, but I'm using github codespaces to manage the repo. As you can see I updated the file but vscode stripped the new line at the end. If it's required I will push an update again.

Comment (Grimler91) [reply packages/pocketbase/build.sh:28]: Tabs should be used as per [coding guidelines](https://github.com/termux/termux-packages/wiki/Coding-guideline). 

Pretty much all other files in the repo end with newline, I don't see why this one should be different

Comment (SunPodder) [reply packages/pocketbase/build.sh]: Fixed

**Pull request merged**